workflow-bot-app/src/main/java/com/symphony/bdk/workflow/versioning/model/VersionedWorkflow.java

```diff
@@ -0,0 +1,73 @@
+package com.symphony.bdk.workflow.versioning.model;
+
+import lombok.ToString;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Lob;
+
+@Entity
+@ToString
+//@EqualsAndHashCode
+public class VersionedWorkflow {
+
+  @EmbeddedId
```

Comment:
i think it s better to simplify the thing, just use field id and version in this model, and add a unique constraint index on both id and version fields, such that the same will be supported by mongodb, otherwise i afraid if we migrate to mongodb one day, this model has to rewrite.